github/provider_test.go

```diff
@@ -71,6 +71,18 @@ func TestProvider_individual(t *testing.T) {
 	individual = true
 }
 `
+
+	conflictingProviderConfig := `provider "github" {
+	individual = true
+}
+`
+
+	neitherProviderConfig := `provider "github" {
+	organization = ""
```

Comment:
I think the more common user scenario, and the one we should probably be testing for, is that they didn't specify organization at all, rather than setting it to an empty string.

Reply:
@paddycarver I agree, however, the rest of the tests need GITHUB_ORGANIZATION defined, so, when running the suite it will already be there, this will override that.  Is there a better way to do this?